sparrow-py/pysrc/sparrow_py/expr.py

```diff
@@ -314,8 +317,19 @@ def _aggregation(op: str, input: Expr, window: Optional[Window]) -> Expr:
     if window is None:
         return Expr.call(op, input, None, None)
     elif isinstance(window, SinceWindow):
-        return Expr.call(op, input, window._predicate, 1)
+        return Expr.call(op, input, window._predicate, None)
     elif isinstance(window, SlidingWindow):
         return Expr.call(op, input, window._predicate, window._duration)
     else:
         raise ValueError(f"Unknown window type {window!r}")
+
+def _collect(input: Expr, max: Optional[int], window: Optional[Window]) -> Expr:
+    """Helper for creating collect expr"""
```

Comment:
Run `nox` to make sure this passes in all lints, etc. I suspect you're going to get an error saying the first sentence needs to be imperative (describe what the function does). Eg. "Create a collect expression" vs. "Helper for ..."